public/index.html

```diff
@@ -4,9 +4,13 @@
     <meta charset="UTF-8">
     <title>Deer</title>
     <link rel="stylesheet" href="../build/bundle.css">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
+
   </head>
   <body>
     <div id="app"></div>
     <script src="../build/bundle.js"></script>
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
```

Comment:
Picking a UI framework needs to be discussed in the Gitter channel. We can't pick one and keep changing it later on. Please ensure that repo owners/moderators have consensus on UI framework.